build.cmd

```diff
@@ -18,6 +18,12 @@ if %errorlevel% equ 0 (
   set outloop=true
 )
 
+echo %* | findstr /i /C:"FloatingTestRuntimeDependencies=true"  1>nul
+if %errorlevel% equ 0 (
+  set TestFixedRuntimeProjectJson=%~dp0src\Common\test-runtime\LatestDependencies\project.json
+  set defaultBuildConfig=/p:Configuration=Windows_NT_Debug
```

Comment:
I am also editing build.cmd so we are going to conflict -- just head's up for one of us.  I also just discovered %ERRORLEVEL% and ERRORLEVEL don't expand correctly when used multiple times in the same CMD file unless ENABLEDELAYEDEXPANSION is set.  You may want to look at #770.